docs/src/troubleshooting.md

```diff
@@ -61,43 +68,29 @@ Another solution could be (at least on Linux) to update the server `PATH` using
 
 ### 1. LSP doesn't start my language server
 
-* Make sure you have a folder added in your Sublime workspace.
-* Make sure the document you are opening lives under that folder.
```

Comment:
Removed as this is no longer the case, we pretty much always start the server. In some cases like LSP-typescript we prevent the start but that gives some feedback in the status bar.